src/biogeochem/CNCStateUpdate1Mod.F90

```diff
@@ -451,15 +518,18 @@ subroutine CStateUpdate1( num_soilc, filter_soilc, num_soilp, filter_soilp, &
             cs_veg%livecrootc_xfer_patch(p)    = cs_veg%livecrootc_xfer_patch(p)   + cf_veg%livecrootc_storage_to_xfer_patch(p)*dt
             cs_veg%deadcrootc_storage_patch(p) = cs_veg%deadcrootc_storage_patch(p)- cf_veg%deadcrootc_storage_to_xfer_patch(p)*dt
             cs_veg%deadcrootc_xfer_patch(p)    = cs_veg%deadcrootc_xfer_patch(p)   + cf_veg%deadcrootc_storage_to_xfer_patch(p)*dt
+           end if !use_matrixcn
             cs_veg%gresp_storage_patch(p)      = cs_veg%gresp_storage_patch(p)     - cf_veg%gresp_storage_to_xfer_patch(p)*dt
```

Comment:
Moving these two lines to above `if(.not. use_matrixcn)then` (lines 511 and 512) might help the readability of the code.

Reply:
Move Line 522 (cs_veg%gresp_storage_patch(p) =...) and 523 to above if(.not. use_matrixcn)then